README.md

```diff
@@ -361,37 +357,36 @@ application developer were to handle this data, they could rely on the same sche
-the different keys to each other, but also to convert their values to a standard format even though
-they all have the same semantic meaning. They all describe when a person was born.
+Take, for example, our previous user model. Right now, each application or service might use their
+own model definitions: one site could choose `birthday`, while another uses `day_of_birth` or
+`birthDay`, to represent the user's birthday. These models might also be of different formats: some
+could be in `YYYY-MM-DD` while others `DD-MM-YYYY`. Despite all of them containing the same semantic
+meaning, custom logic would have to be written to not only handle the mapping between the different
+keys, but also to convert their values to a standard format.
 
-JSON-LD solves these problems by introducing a `context` to each data model allowing for:
+JSON-LD's `context` solves these problems by allowing for:
 
-1. A unified mapping of keys by complying to base schemata
-1. Value-validation on a [primitive data types level](https://www.w3.org/TR/rdf11-concepts/#section-Datatypes)
+1. A unified mapping of keys that comply to base schemata; and
+1. Value validation for [primitive data types](https://www.w3.org/TR/rdf11-concepts/#section-Datatypes).
 
 
-Going back to the Andy Warhol example, we need to explain how JSON-LD magically maps our
-self-defined keys (`givenName`, `familyName` and `birthDate`) to the properties of schema.org's
-`Person`. If you look at schema.org's `Person` definition, you'll see we didn't choose random key
-names. Names were already part of the definition. In this case, a JSON-LD parser can automatically
-map and execute validation against these properties by using the schema definition.
+To see how `context` achieves this, we need to explain how JSON-LD magically maps our example's
+self-defined keys (`givenName`, `familyName` and `birthDate`) to their matching properties on the
+`Person` schema. If you look at the [`Person` definition](http://schema.org/Person), you'll notice
+that we didn't choose random keys–these keys were already part of the schema definition. Because of
+this, JSON-LD parsers are able to automatically map and validate our example model's properties
+against their schema definitions.
 
-For more clarity, let's see how a JSON-LD parser would look at this example:
+For more clarity, let's see how a JSON-LD parser would look at our example:
 
 1. Notice `@context` contains `http://schema.org/Person`
 1. `GET http://schema.org/Person`
-1. For each of the user-defined keys, check if they map to any keys provided in the schema
-    1. If this is the case, traverse the schema until a leaf node (the JSON-LD specification calls
-       this an `identifying blank node`) is found
+1. For each of the model's keys, check if they map to any keys provided in the schema
+    1. For each matched key, traverse the schema until the matching URI is found
```

Comment:
"For each matched key, traverse the schema until the matching URI is found"

We're kind of removing a lot of information in this edit. Compare to:

"If this is the case, traverse the schema until a leaf node (the JSON-LD specification calls this an `identifying blank node`) is found".

Since we're talking about a JSON-LD parser here, I prefer the old version.